package.json

```diff
@@ -95,6 +95,7 @@
     "json-stable-stringify": "1.0.1",
     "key-mirror": "1.0.1",
     "keymaster": "1.6.2",
+    "libphonenumber-js": "0.4.27",
```

Comment:
Now it's not the time to think about this... we should migrate the logic as-is first, and then improve upon it. If you're worried about the size, at least this will only affect the `woocommerce` bundle.